src/Commands/Postgres/Table/GetSchemaCommand.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using AzureMcp.Arguments.Postgres.Table;
+using AzureMcp.Models.Argument;
+using AzureMcp.Models.Command;
+using AzureMcp.Services.Interfaces;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol.Server;
+
+namespace AzureMcp.Commands.Postgres.Table;
+
+public sealed class GetSchemaCommand(ILogger<GetSchemaCommand> logger) : BasePostgresCommand<GetSchemaArguments>(logger)
+{
+    private readonly Option<string> _tableOption = ArgumentDefinitions.Postgres.Table.ToOption();
+    protected override string GetCommandName() => "get-schema";
+
+    protected override string GetCommandDescription() =>
+        "Retrieves the schema of a specified table in a PostgreSQL database.";
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+        command.AddOption(_tableOption);
+    }
+
+    protected override void RegisterArguments()
+    {
+        base.RegisterArguments();
+        AddArgument(CreateTableArgument());
+    }
+
+    protected override GetSchemaArguments BindArguments(ParseResult parseResult)
+    {
+        var args = base.BindArguments(parseResult);
+        args.Table = parseResult.GetValueForOption(_tableOption);
+        return args;
+    }
+
+
+    [McpServerTool(Destructive = false, ReadOnly = true)]
+    public override async Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        try
+        {
+            var args = BindArguments(parseResult);
+            args.Validate();
```

Comment:
an you mark the arg as required?  BindArguments takes care of validating the required params.

Reply:
I think we should remove all those Validate() methods.